packages/client/hmi-client/src/api/api.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import useAuthStore from '../stores/auth';
+
+const API = axios.create({
+	baseURL: '/api'
+});
+
+// Hook in bearer tokens
+// See
+// - https://medium.com/swlh/handling-access-and-refresh-tokens-using-axios-interceptors-3970b601a5da
+API.interceptors.request.use((config) => {
```

Comment:
do we foresee any need to eject this interceptor in the future? if so then this should be saved in a const that is added later and can be removed as needed

Reply:
I don't know what you mean by `eject this interceptor`

Reply:
sorry `axios` terminology, `eject` simply means remove or unsubscribe
ie.
```
const myInterceptor = axios.interceptors.request.use(function () {/*...*/});
axios.interceptors.request.eject(myInterceptor);
```

I don't think we will need to but just covering the bases.

Reply:
Got it! Yeah I don't think we will need it, but will keep it mind, thanks!